component/alertmanager.jsonnet

```
@@ -0,0 +1,101 @@
+local com = import 'lib/commodore.libjsonnet';
+local kap = import 'lib/kapitan.libjsonnet';
+local kube = import 'lib/kube.libjsonnet';
+local inv = kap.inventory();
+local params = inv.parameters.rancher_monitoring;
+local namespace = params.namespace;
+local instance = params.alertmanagerInstance;
+local name = 'alertmanager-' + instance;
+
+local labels = {
+  'app.kubernetes.io/name': 'alertmanager',
```

```suggestion
  'app.kubernetes.io/name': name,
```

Not sure about this one.

---

According to https://syn.tools/syn/references/labels-guide.html which has the information from the Kubernetes docs:

> `app.kubernetes.io/name` -- The name of the application -- `steward`

Given that information, `alertmanager` should be the correct value for the `app.kubernetes.io/name` label